### ISSUE TITLE
Consistent merge of user and app config

### ISSUE THREAD

**Issue by tarkh** [aristocratos/btop#130]
**Is your feature request related to a problem? Please describe.**
If I create `.config/btop/btop.conf` with only options that I do want to alter, i.e
```
color_theme = "/usr/share/btop/themes/dracula.theme"
truecolor = False
```
then `btop` will create full config, while overriding `color_theme` with Default, but keeping `truecolor` boolean as False.
If I preset two boolean options in my pre-configuration, then `btop` will override firs with default value, but keep last one with custom.

**Describe the solution you'd like**
As possible logical solution `btop` can consistently merge options from local config (`.config/btop/btop.conf`) on top of all that exist internally in app itself. This way we can preset configs with only those options that we want to alter. The rest will be merged by app on first run. This may be useful while doing some custom specific distributions.

**Comment by aristocratos** [comment]
@tarkh 
The config parser was expecting the first line to be a commented line with current version.

Fixed in latest commit.

**Comment by aristocratos** [comment]
Fixed in [v1.0.23](https://github.com/aristocratos/btop/releases/tag/v1.0.23)
